bbridge_client.py

```diff
@@ -48,19 +49,22 @@ def __init__(self, token, host_url="http://bbridgeapi.cloudapp.net/v1"):
         self.__sentiment_analysis_url = "{}/nlp/sentiment".format(host_url)
         self.__name_entity_recognition_url = "{}/nlp/ner".format(host_url)
 
-    def response(self, request_id, inner_type=None):
+    def response(self, request_id, return_type=None):
         """
         :type request_id: str
-        :type inner_type: type | None
+        :type return_type: entity.response.bbridge_entity.BBridgeEntity | None
```

Comment:
wrong type annotation. `response` requires some class extends `BBridgeEntity` not object of `BBridgeEntity `